src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const handlerFunction = (text, classIn) => {
+  const div = document.createElement('div');
+
+  div.dataset.qa = 'notification';
+  div.className = classIn;
+  div.textContent = text;
+
+  document.body.append(div);
+};
+
+let leftClick = false;
```

Comment:
Add a brief comment explaining the purpose of 'leftClick' and 'rightClick' variables.